frame_metadata/FrameMetadata.ts

```diff
@@ -0,0 +1,44 @@
+import * as $ from "../deps/scale.ts"
+
+export interface FrameMetadata {
```

Comment:
This processed metadata shape corresponds to v14 –– perhaps instead of `raw/v14.ts`, it might make more sense to have a `v14` directory, within which we could place both the raw codec and normalized declaration, along with the transformation from raw to normalized?

Reply:
This processed metadata shape is for *all* versions. As new versions come out, we'll add more (likely optional) fields to this, and different raw frame metadata versions will all transform into this one shape.